### PR TITLE
Rebase latest commits

### PR DESCRIPTION
**Pull request by nuxzero**
**1. Objective**

This PR purpose to merge the latest commits from the `master-backup` branch to the `master` branch.

**2. Description of change**

Merge these commits to the `master` branch. And fix the unit tests.

```
1351cb1 Upgrade shadowJar version (#96)
410386a Generate payload methods (#95)
96ec5db Remove base Model from PlatformFee (#94)
726bb8f Auto generate codes (Models, Enums and Request Builders) (#92)
b46705d Added public modification to API_VERSION. (#93)
7ce0df4 Fix incorrect field types and make it matches with latest API (2019) (#90)
a8209f8 Deprecated Transfer model DestroyRequestBuilder  (#91)
58c23a9 Upgraded Gradle wrapper to version 5.6.1 (#89)
3d61958 Field type corrections (#88)
46bb749 Removed certificate pinning from Client. (#87)
```

**3. Quality assurance**

All unit tests shall be passed.

> **Note**
>
> This PR looks huge but not really. The changes are only 3 last commits you can start review this PR from those commits.
>
> ```
> a8fc972 test: fix tests
> a274504 fix: add missing PromptPay type
> 3794074 fix: fix conflicts
> ```

**4. Operations impact**

N/A

**5. Business impact**

N/A

**6. Priority of change**

Normal

### CODE REVIEW

**Comment by rezigned** [comment]
Can we extract #92 #94 #95? (I'll put them in v4.0 - #106)

File: src/main/java/co/omise/models/Account.java [hunk 1]
```diff
@@ -5,50 +5,72 @@
...
+        return this.currency;
+    }
+
+    public void setCurrency(String currency) {
+        this.currency = currency;
     }
 
     public String getEmail() {
-        return email;
+        return this.email;
```
Comment (nikolack) [inline src/main/java/co/omise/models/Account.java:36]: I see that `this.` is added to every getter. Is this done with a purpose or is it just some coding style?

File: src/main/java/co/omise/models/Balance.java [hunk 2]
```diff
@@ -4,49 +4,69 @@
...
-    public void setTransferable(long transferable) {
-        this.transferable = transferable;
+    public String getLocation() {
+        return this.location;
+    }
+
+    public void setLocation(String location) {
+        this.location = location;
+    }
+
```
Comment (nikolack) [inline src/main/java/co/omise/models/Balance.java:38]: I hate these get/set methods in Java more than anything. Why not just make everything public?
Sorry, no need to change anything, I'm just ranting :D

**Comment by nuxzero** [comment]
@kung I removed those 3 commits and removed CircleCI's config. Could you review again please?

File: src/main/java/co/omise/models/Balance.java (hunk 2 above)
Comment (nuxzero) [reply src/main/java/co/omise/models/Balance.java]: @nikolack Yeah, it doesn't convenient. But I think we should follow Java code conventional. 🙂

File: src/main/java/co/omise/models/SourceType.java [hunk 3]
```diff
@@ -3,83 +3,56 @@
...
+    InternetBankingBay,
+    @JsonProperty("internet_banking_bbl")
+    InternetBankingBbl,
+    @JsonProperty("internet_banking_ktb")
+    InternetBankingKtb,
+    @JsonProperty("internet_banking_scb")
+    InternetBankingScb,
     @JsonProperty("paynow")
     Paynow,
-    @JsonProperty("points_citi")
```
Comment (rezigned) [inline src/main/java/co/omise/models/SourceType.java:39]: Forgot to put this one back 😆

Comment (nuxzero) [reply src/main/java/co/omise/models/SourceType.java:39]: Changed.

**Pull request merged**